src/app/legacy/containers/Header/index.jsx

```diff
@@ -46,11 +46,14 @@ const HeaderContainer = ({
   scriptSwitchId = '',
   renderScriptSwitch = true,
 }) => {
-  const { pageType, isAmp, isApp } = useContext(RequestContext);
+  const { pageType, isAmp, isApp, pathname } = useContext(RequestContext);
   const { service, script, translations, dir, scriptLink, lang, serviceLang } =
     useContext(ServiceContext);
   const { skipLinkText } = translations;
 
+  const isOptimoId =
```

Comment:
You may be able to use the `pageType` rather than checking for the Optimo ID pattern. `pageType` should return `article` for native Optimo articles.